CONTRIBUTING.md

```diff
@@ -1,25 +1,28 @@
 # Contributing
 
-
 ## Development Setup
+
 ### Prerequisite Tools
 
 - [Git](https://git-scm.com/)
-- [Go (at least Go 1.20)](https://golang.org/dl/)
+- [Go (at least Go 1.21.3)](https://golang.org/dl/)
 
 ### Environment
 
 - Fork the repository.
 - Clone your forked repository locally.
 - We use Go Modules to manage dependencies, so you can develop outside your `$GOPATH`.
 - We use [golangci-lint](https://github.com/golangci/golangci-lint) to lint our code, you can install it locally via `make setup`.
+
 ### Building
+
 - Enter the provider directory
 - Run `make tools` to install the needed tools for the provider
-- Run `link-git-hooks` to install githooks 
-- Run `make build` to build the binary in the `./bin` directory: 
+- Run `link-git-hooks` to install githooks
+- Run `make build` to build the binary in the `./bin` directory:
 - Use the local provider binary in the `./bin` folder:
-  - Create the following `dev.trfc` file inside your directory 
+  - Create the following `dev.trfc` file inside your directory
```

Comment:
```suggestion
 - Create the following `dev.trfc` file inside your directory
```
this file seems like autoformated but since here

Reply:
reformat undone as it's not the purpose of this PR